src/traces/scattersmith/attributes.js

```diff
@@ -23,9 +23,9 @@ module.exports = {
     },
 
     text: scatterAttrs.text,
-    texttemplate: texttemplateAttrs({editType: 'plot'}, {
-        keys: ['re', 'im', 'text']
-    }),
```

Comment:
I think we should keep `texttemplate` feature.

Reply:
I've restored the feature in 00630415eff3d4fd03812770e750e961e124e61a, but I don't think it is hooked up properly.